src/main/java/bio/terra/janitor/service/iam/AuthenticatedUserRequest.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.janitor.service.iam;
+
+import bio.terra.janitor.common.exception.ApiException;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Optional;
+import java.util.UUID;
+
+public class AuthenticatedUserRequest {
```

Comment:
Janitor uses autovalues a lot. I uses a normal class is to follow WM and TDR's convension. Shall I change to AutoValue? One problem in aothValue I recently we saw is  AutoValue works well with Gson, but for Jackson, it needs some extra setup. Having getter and setter as regular class like that makes Jackson easier to serialize a class.(I am not sure if we need to serialize this class through). 
CRL uses Gson. But Janitor uses Jackson because that is the what Spring's Openapi generator and Stairway uses. 
Not sure what is the common conversion when choosing java libraries for MCTerra? Personally I likes Gson because it is simpler and get ride of the annoying checked exception - `JsonProcessingException`

Reply:
Ah, interesting tradeoffs here. IMO it hinges on whether we expect this class to be shared w/ other services in a common library. If so, we should probably be as plain as possible. But if not, then using autovalue sounds reasonable.

Re: the GSON vs. Jackson question, the swagger & stairway dependencies on Jackson probably make it overall the right decision to stick with Jackson everywhere. Hopefully not too much of a drag :) and I'll make sure to add some note on the tradeoffs involved into our tech stack documentation.